src/main/java/com/endava/soa_5ed/task3_oop/Cat.java

```diff
@@ -0,0 +1,85 @@
+package com.endava.soa_5ed.task3_oop;
+
+public class Cat extends Feline implements Tail {
```

Comment:
To use a logger. 
You simply need to use @Slf4j annotation behind your class 
or inside the class declare a private final Logger log = LoggerFactory.getLogger(this.getClass);